components/ReviewForm/index.jsx

```diff
@@ -36,11 +37,12 @@ const Button = styled.button`
   }
 `;
 
-const ReviewForm = ({ onSubmit }) => {
+const ReviewForm = ({ onSubmit, rating, setRating }) => {
   return (
     <FormWrapper onSubmit={onSubmit}>
       <Label htmlFor="review">Add Review</Label>
       <InputText type="text" id="review" name="review" />
```

Comment:
I recommend to make the input `required` so the user gets visual feedback when he tries to submit the form with just a star-rating.